### PR TITLE
Feature/908 supporting different screen sizes

### PR DESCRIPTION
**Pull request by Joeu**
### What does this pull request aim to achieve? 💯

Unit changing from px to rem units, allowing us to change elements proportions by just changing the value of the font-size in the GlobalStyles file.

_Please be concise and link any related issue(s):_ #908 

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [ ] This branch is rebased/merged with the **latest master**.
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Comment by idjevm** [comment]
Let's open this up, besides the changes that @BMariscal pointed out, there just might be little things that we will need to fix here and there but Loooks already WAAAY better of how it is right now

**Comment by Joeu** [comment]
Requested changes applied, @idjevm  and @BMariscal .

File: client/src/components/Feed/PostCard.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ const PostCard = styled(Card)`
         margin-top: -1.8rem;
 
         .status-icon {
+          width: 1rem;
```
Comment (BMariscal) [inline client/src/components/Feed/PostCard.js:91]: The size of the blue button next to the country name is too big atm but this should fix it: 
```suggestion
          width: fit-content;
```

**Comment by BMariscal** [comment]
When you click on the "Create Post" button, the icons inside the buttons need to be resized and realigned.

**Comment by BMariscal** [comment]
If you go the user's profile page, the "Create Post" button is still too small:

**Comment by BMariscal** [comment]
Drop down arrows within the visibility buttons, in this case 'City' and 'Month' need to be fixed.

**Comment by Joeu** [comment]
Just pushed changes related to the modals, I'm still counting on you to help me cover the whole layout, @idjevm and @BMariscal .  As we have no specs for higher resolutions, we will keep finding places where changes will be needed.
The main goal of my changes is to change proportions by changing the font-size in GlobalStyles.

**Comment by BMariscal** [comment]
Thanks for all the work you've done here @Joeu 
My screen resolution is just "1440 x 900" 

The icons inside the Create Post modal are too small:


But this is how you can fix them: 

add a class, in this case I called it `.icon`, to the images and unset the height and width.

**Comment by BMariscal** [comment]
Also this still too small
 

Please rebase and fix the merge conflicts. Thanks for all your work!

**Comment by BMariscal** [comment]
> We are almost there, just a few more changes though:
> image
> 
> image
> 
> image
> 
> image
> 
> Org Profile:
> image
> 
> I think it needs a bit more work, in org profile, post tags across orgs / profile / feed . And also the same buttons and arrows a previously mentioned by Bri, let me know I can slack you and show you more xD

hmm, the sign in button and the tags look good to me, even on 2560 x 1440 resolution. What resolution are you testing these with? 

The org profile page, not the user profile, still needs work. Why does the same button shared across the app not have the same styling? These should be set in one place, GlobalStyles, and inherited everywhere else.

**Comment by Joeu** [comment]
I agree with you about the button, JV, we need to dig more into that to see why we have WizardSubmit and SubmitButton.

**Comment by Joeu** [comment]
Check again the tags on the Feed, please.

**Comment by Joeu** [comment]
My point is that we should push the code and fix those small issues later, the main goal was to change the units from pixels to rem. But I agree with you and I can say that more tweaks like that will appear as we go through application flow.

**Pull request merged**